### DISCUSSION TITLE
Win 11 23H2

### DISCUSSION THREAD

**Discussion by papapingouin-be** [jiangcuo/Proxmox-Port#60] [Q&A]
Hello, this is a great project, and thank you already for everything that works.

I'm trying to install Windows, and I managed to boot from the ISO CD-ROM, but I encountered an error page. Do you have any ideas for what to do next?
![image](https://github.com/jiangcuo/Proxmox-Port/assets/82971316/e9131c19-970f-4de9-a8f3-e1b156ae1279)
![image](https://github.com/jiangcuo/Proxmox-Port/assets/82971316/d8508c70-05ca-4e48-b51f-ec2bf1e61e53)

my win 11 iso from :https://uup.rg-adguard.net/
Thanks in advance,
Laurent

**Comment by gmenezes-sistemas11** [comment]
Can u confirm if that iso is in the right architecture? aarch64(arm64) for RPI4?
